src/main/java/Alex/Alex.java

```diff
@@ -0,0 +1,161 @@
+package Alex;
+
+import Alex.exception.DukeException;
+import Alex.manager.TaskManager;
+import Alex.task.Deadline;
+import Alex.task.Event;
+import Alex.task.Task;
+import Alex.task.Todo;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class Alex {
+    public static void main(String[] args) {
+        TaskManager taskManager = new TaskManager();
+        Scanner myObj = new Scanner(System.in);
+        greeting();
```

Comment:
Try to keep the abstractions on the same level. Do take a look at how SLAP should be done :>